ipnsfs/system.go

```diff
@@ -0,0 +1,285 @@
+package ipnsfs
```

Comment:
agreed. `ipnsfs` was just a quick working name i gave it so i could get to coding. I also dont know if I like it being in the root namespace, but not sure where else it should go.